packages/server/test/e2e/3_issue_1987.ts

```diff
@@ -0,0 +1,21 @@
+const e2e = require('../support/helpers/e2e')
+
+describe('e2e issue 1987', () => {
+  e2e.setup({
+    servers: [{
+      port: 3434,
+      static: true,
+    },
+    {
+      port: 4545,
+      static: true,
+    }],
+  })
+
+  // https://github.com/cypress-io/cypress/issues/1987
+  // before/after hooks should not be rerun on top navigation
```

Comment:
you mention after hooks, but these aren't really tested in the test

```suggestion
  // before hooks should not be rerun on top navigation
```